packages/superset-ui-connection/src/callApi/parseResponse.ts

```diff
@@ -0,0 +1,27 @@
+const PARSERS = {
+  json: (response: Response): Promise<JsonResponse | any> =>
```

Comment:
the final line of this file would not compile unless I added `any` here, I thiiink it's due to a possible error but am not sure. It's weird to me that the return type of the `parseResponse` Promise can not include `any` 🤔

Reply:
ah, that makes more sense 👌